etrago/appl.py

```diff
@@ -112,12 +112,15 @@
     "network_clustering_ehv": {
         "active": False,  # choose if clustering of HV buses to EHV buses is activated
         "busmap": False,  # False or path to stored busmap
+        "interest_area": False,  # False, path to shapefile or list of nuts names of not cluster area
     },
     "network_clustering": {
         "active": True,  # choose if clustering is activated
         "method": "kmedoids-dijkstra",  # choose clustering method: kmeans or kmedoids-dijkstra
         "n_clusters_AC": 30,  # total number of resulting AC nodes (DE+foreign)
         "cluster_foreign_AC": False,  # take foreign AC buses into account, True or False
+        "interest_area": False,  # False, path to shapefile or list of nuts names of not cluster area
```

Comment:
Here, I would write "of area that is clustered to the number of clusters specified through "cluster_interest_area"". For "n_clusters_AC" above I would write an explanation, that the "interest_area" is excluded (DE+foreign-interest_area), or is it not?